docs/framework-integration/angular.md

```diff
@@ -407,6 +408,16 @@ first file you should import in your Angular project.
 If `true`, Angular components will import and define elements from the `dist-custom-elements` build, rather than `dist`. For more information
 on using the `dist-custom-elements` output for the Angular proxies, see the [FAQ answer below](#do-i-have-to-use-the-dist-output-target).
 
+### includeSingleComponentAngularModules
+
+**Optional**
+
+**Default: `false`**
+
+**Type: `boolean`**
+
+If `true`, an Angular module will be generated for each component. This option requires that `includeImportCustomElements` is set to `true`.
```

Comment:
Yeah I think that makes sense for now 👍 the framework wrapper docs should really be a separate docs plugin instance (within the same docs site). This would allow us to version the framework wrapper docs independent of the Stencil docs. This would also introduce the version dropdown for the framework wrapper version instead of Stencil's.